src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -687,16 +687,18 @@ class Project: ADOSVTBase
     
     hidden [ControlResult] CheckSecureFilesPermission([ControlResult] $controlResult) {
         # getting the project ID
-        $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
-        $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/$($projectId)/_apis/distributedtask/securefiles?api-version=6.1-preview.1"
         try {
+            $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
+            $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/$($projectId)/_apis/distributedtask/securefiles?api-version=6.1-preview.1"
             $response = [WebRequestHelper]::InvokeGetWebRequest($url);
             # check on response object, if null -> no secure files present
             if(([Helpers]::CheckMember($response[0],"count",$false)) -and ($response[0].count -eq 0)) {
                 $controlResult.AddMessage([VerificationResult]::Passed, "There are no secure files present.");
             }
             # else there are secure files present
-            elseif((-not ([Helpers]::CheckMember($response[0],"count"))) -and ($response.Count -gt 0)) {
+            #WHY elseif and not else
```

Comment:
Is this something you want to discuss? If yes, do not check in this comment in code

Also test all the scenarios of control eval to see nothing breaks due to change from else if to else